src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -53,6 +53,8 @@
 
 #include "application_manager/mock_app_service_manager.h"
 #include "resumption/last_state_impl.h"
+#include "resumption/last_state_wrapper.h"
+#include "resumption/last_state_wrapper_impl.h"
```

Comment:
@YarikMamykin do you really need include both headers?

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/3dcb90541c1ee0310a396d2564a95eba3e12237e